src/last-transactions.ts

```diff
@@ -75,26 +83,10 @@ export const lastTransactions: APIGatewayProxyHandler = async (event): Promise<a
     };
   }
 
-  const parsedTransactionCountLimit = transactionCountLimit ? parseInt(transactionCountLimit) : NaN;
-  const queryTransactionCountLimit = !isNaN(parsedTransactionCountLimit)
-    ? parsedTransactionCountLimit
-    : DEFAULT_TRANSACTION_COUNT;
+  const queryTransactionCountLimit = DEFAULT_TRANSACTION_COUNT;
 
   const operation = async () =>
-    db.query(
-      `
-    SELECT 
-    event_data -> 'blockNumber' as "blockNumber", 
-    event_data -> 'parsedLog' as "parsedLog",
-    event_data -> 'topics' as "topics",
-    event_data -> 'transactionHash' as "transactionHash"
-    FROM dapi_events 
-    WHERE chain = $1 AND 
-    event_data->'parsedLog'->'args'->> 0 = $2
-    ORDER by block DESC LIMIT $3;
-  `,
-      [queryChainId, queryBeaconId, queryTransactionCountLimit]
-    );
+    db.query(lastTransactionsQueryTemplate, [queryChainId, queryBeaconId, queryTransactionCountLimit]);
```

Comment:
It's better to inline the `lastTransactionsQueryTemplate` since it could go out of sync with the arguments (since it's defined way above the usage).